bors.toml

```
@@ -1 +1 @@
-status = ["continuous-integration/travis-ci/push"]
+status = ["build / Build and Test (push)"]
```

Just guessing, but maybe just

```suggestion
status = ["build / Build and Test"]
```

The thing in parens is just the trigger (see in the PR checks where it is `(pull request)`.

---

If https://www.christopherbiscardi.com/integrating-bors-with-github-actions is to believe, I think it should be

```suggestion
status = ["Build and Test"]
```

---

Lets see

---

That was it

---

is this back to before or is github's ui tricking me?